setup.cfg

```diff
@@ -2,7 +2,7 @@
 universal = 1
 [flake8]
 application-import-names = orderable
-ignore = D100,D101,D102,D203,D204
+ignore = D100,D101,D102,D104,D105,D203,D204
```

Comment:
Yeah, no rush.  (We'll want to remove it in our other projects as well, if everyone's on board with the idea.)